src/renderer/ManualReport.tsx

```diff
@@ -123,8 +131,9 @@ export default function ManualReport({
     for (let n = 1; n <= entrant1Score + entrant2Score; n += 1) {
       gameData.push({
         gameNum: n,
-        winnerId:
-          n <= entrant1Score ? selectedSet.entrant1Id : selectedSet.entrant2Id,
+        winnerId: (n <= entrant1Score
+          ? selectedSet.entrant1Id
+          : selectedSet.entrant2Id) as number,
```

Comment:
StartggGame is an API type. If we can't do this without an unsafe cast, let's split up the logic for sgg and parrygg. It's ok if the logic is mostly duplicated